### ISSUE TITLE
ISO3166-2 missing for Namibia (NA)

### ISSUE THREAD

**Issue by jimjam-slam** [open-numbers/ddf--gapminder--population#9]
Namibia has the two-digit ISO3166-2 code 'NA'. It doesn't appear in `ddf--entitites--geo--country.csv`. I'm guessing it got turned from a string into an actual NA value somewhere down the line and has been silently lost :/

**Comment by jheeffer** [comment]
Confirmed, thanks for finding this @rensa! It was removed at some point in this commit on the geo entity domain repo:

https://github.com/open-numbers/ddf--gapminder--geo_entity_domain/commit/a4ad8cc60e50f8668dece793f1aa88c87d43bbbe#diff-571b07c968a63fb385551ea2c2e7e996L162

@semio can you look at why this happened? Possibly `NA` was seen as Not Available value like @rensa suggests.

**Comment by semio** [comment]
@jheeffer yes, I checked the documents and confirmed pandas will treat 'NA' as not available by default. I will make sure to check this and avoid such issue in etl scripts. Thanks!
